modules/web/js/ballerina/ast/connector-declaration.js

```diff
@@ -250,6 +250,11 @@ define(['lodash', './node', '../utils/common-utils'], function(_, ASTNode, Commo
         return expression;
     };
 
+    var shouldSkipPackageName = function(packageName) {
+        return _.isUndefined(packageName) || _.isNil(packageName) ||
```

Comment:
No need to check with both isUndefined and  isNil. isNil checks for both null and undefined